### PR TITLE
Assembly: Fix logic error when omitting members

### PR DESCRIPTION
**Pull request by tokusanya**
Also added a small format change

### CODE REVIEW

**Comment by gsjaardema** [comment]
I don't see the logic error change; only two format changes...  Is something missing in the commit?

**Comment by tokusanya** [comment]
Line 775 had a logic error where the GroupingEntity is not a nullptr but omitted ...this would cause it to throw which was not correct. It should only throw if the GroupingEntity is null

File: packages/seacas/libraries/ioss/src/exodus/Ioex_BaseDatabaseIO.C [hunk 1]
```diff
@@ -771,10 +771,11 @@ namespace Ioex {
             assem->add(ge);
             num_added_entities++;
           }
-          else {
+
+          if (ge == nullptr) {
```
Comment (gsjaardema) [inline packages/seacas/libraries/ioss/src/exodus/Ioex_BaseDatabaseIO.C:775]: I think the overall logic here would be simpler if this `ge == nullptr` test were put right after the `auto *ge = ...` line.  Right now we have a test: 
```
 if (ge != nullptr && !Ioss::Utils::block_is_omitted(ge)) {
```
and then a test: 
```
 if (ge == nullptr) {
```
If the second test was first, then the first test would just be:
```
if (!Ioss::Utils::block_is_omitted(ge))
```

**Review by gsjaardema — COMMENTED** [review]
Is ok to merge as is, but I think the logic is easier to read if the two tests are reordered.

File: packages/seacas/libraries/ioss/src/exodus/Ioex_BaseDatabaseIO.C (hunk 1 above)
Comment (tokusanya) [reply packages/seacas/libraries/ioss/src/exodus/Ioex_BaseDatabaseIO.C:775]: That sounds good

**Pull request merged**